src/app/services/constants.ts

```diff
@@ -778,9 +778,8 @@ export enum EventStatusFilter {
 }
 export enum EventTypeFilter {
     "All events" = "all",
-    "Student events" = "student",
-    "Teacher events" = "teacher",
-    "Online tutorials" = "virtual",
+    "Booster" = "booster",
```

Comment:
Just double-checking that this is correct: it looks like 3 types have been removed and 2 new added in this filter. One of the removed types is `student`, but in the other files that you've modified in this PR you've left links to `/events?types=student`. I'm just mentioning this to ensure that this is all correct.

Reply:
Removed the `?types=student` from event links, since all events are now student events!

Reply:
There is one rogue component (`TeacherFeatures`) that is still using that type of link. But that component does not seem to be used anywhere. Maybe that means that it can be fully removed.

Reply:
I'm in favour of fully removing it but pending decision as part of this ticket: https://github.com/isaaccomputerscience/isaac-cs-issues/issues/100

So for now I left it alone - if we decide to start using TeacherFeatures it's going to need some work anyway as it was intended just for Physics use.

Reply:
That makes sense, thanks. In that case, you can leave `TeacherFeatures`, but I still think that the `events?types=student` link inside it needs to be updated.

Reply:
That's adjusted now, removed the /events?types=teacher also, since teacher events no longer exist